doc/crud.md

```diff
@@ -53,6 +72,9 @@ extension data such as indexes (see [index-query.md](doc/index-query.md)):
 BOOL success = [manager deleteDatastoreNamed:@"my_datastore"
                                        error:&error];
 ```
+```swift
+try manager.deleteDatastoreNamed("my_datastore")
```

Comment:
What happens to the BOOL return value here? Should this be `let success = try ...`